jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java

```diff
@@ -203,6 +206,27 @@ public static void setupEnv() throws IOException {
         serverConfigFile.deleteOnExit();
         Utils.obtainServerConfig();
     }
+    
```

Comment:
This. stuff was all being haphazardly cleaned up.  So concentrated it & clean up things in the right order.